.github/workflows/ci.yml

```diff
@@ -0,0 +1,38 @@
+name: CI Python 3 on Linux
+
+on: [push, pull_request]
+
+jobs:
+  test:
+
+    strategy:
+      max-parallel: 4
+      matrix:
+        platform: [ubuntu-latest]
```

Comment:
what other platforms can we test on?

Reply:
I tried using

```yaml
 platform: [ubuntu-latest, macos-latest, windows-latest]
```

however, `macos-latest` doesn't support Python 3.7 at the moment, and `windows-latest` has issues with PyTorch. So given our current requirements we can only use the Linux virtual environments to test.

Reply:
Though it would be worth looking at trying to fix the Windows build.